File: app/src/main/java/com/ujujzk/vrgweather/data/api/OpenWeatherApi.java
package com.ujujzk.vrgweather.data.api;


import com.ujujzk.vrgweather.model.openweather.Example;

import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Single;


public interface OpenWeatherApi {

    @GET("/data/2.5/forecast")
    Single<Example> getWeather(
            @Query("q") String cityName,
            @Query("appid") String appId,
            @Query("units") String units,
            @Query("lang") String lang
    );
}

File: networking/src/main/java/com/ujujzk/networking/weather/OpenWeatherApi.kt
package com.ujujzk.networking.weather


import com.ujujzk.networking.weather.model.Example
import io.reactivex.Single

import retrofit2.http.GET
import retrofit2.http.Query


interface OpenWeatherApi {

    @GET("/data/2.5/forecast")
    fun getWeather(
            @Query("q") cityName: String,
            @Query("appid") appId: String,
            @Query("units") units: String,
            @Query("lang") lang: String
    ): Single<Example>
}
